pkgs/by-name/ne/networkmanager-ssh/package.nix

```
@@ -0,0 +1,65 @@
+{
+  stdenv,
+  lib,
+  autoreconfHook,
+  fetchFromGitHub,
+  pkg-config,
+  networkmanager,
+  libsecret,
+  gtk3,
+  gtk4,
+  gnome,
+  glib,
+  libnma,
+  libnma-gtk4,
+  intltool,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "NetworkManager-ssh";
+  version = "1.4.2";
+
+  src = fetchFromGitHub {
+    owner = "danfruehauf";
+    repo = "NetworkManager-ssh";
+    rev = "tags/${version}";
+    sha256 = "sha256-ExCU22V4fYuFXW/HqJ39+PbYykXu4rpk8+3/hg9KTMo=";
+  };
+
+  patches = [
+    ./fix-paths.patch
+  ];
```

```suggestion
  postPatch = ''
    substituteInPlace src/nm-ssh-service.c \
      --replace-fail /usr/bin/ssh ${lib.getExe openssh} \
      --replace-fail /usr/bin/sshpass ${lib.getExe sshpass}
  '';
```
Nix can be used outside of NixOS, thus you should never use NixOS paths outside of `nixos/modules`